larpandoracontent/LArHelpers/LArPcaHelper.cc

```diff
@@ -16,19 +17,20 @@ using namespace pandora;
 namespace lar_content
 {
 
-void LArPcaHelper::RunPca(const CaloHitList &caloHitList, CartesianVector &centroid, EigenValues &outputEigenValues, EigenVectors &outputEigenVectors)
+template <typename T>
+void LArPcaHelper::RunPca(const T &t, CartesianVector &centroid, EigenValues &outputEigenValues, EigenVectors &outputEigenVectors)
 {
-    CartesianPointVector pointVector;
+    CartesianPointDoublePairVector cartesianPointDoublePairVector;
 
-    for (const CaloHit *const pCaloHit : caloHitList)
-        pointVector.push_back(pCaloHit->GetPositionVector());
+    for (const auto &iter : t)
```

Comment:
Alternative name to iter? Maybe e.g. positionValue or similar. Previously point.